File: OfficeFinderBackend/OfficeFinderBackend/Models/Office.cs
using System.Collections.Generic;

namespace OfficeFinderBackend.Models
{
   public class Office
   {
      public long Id { get; set; }
      public string Name { get; set; }
      public string GeoJson { get; set; }
      public IEnumerable<GeoPointString> GeoPointStrings { get; set; }
   }
}

File: OfficeFinder/app/src/main/java/sk/stuba/fiit/officefinder/models/Office.kt
package sk.stuba.fiit.officefinder.models

import com.google.gson.annotations.SerializedName

class Office(
        @SerializedName("name")
        val name: String,
        @SerializedName("id")
        val id: Long,
        @SerializedName("geoPointStrings")
        val geoPointStrings: List<GeoPointString>
)
